app/scripts/components/analysis/results/index.tsx

```diff
@@ -123,17 +123,32 @@ export default function AnalysisResults() {
     };
   }, [start, end, datasetsLayers, aoi]);
 
-  if (errors?.length) {
-    return <Navigate to={thematicAnalysisPath(thematic)} replace />;
-  }
-
   const analysisParamsQs = analysisParams2QueryString({
     start,
     end,
     datasetsLayers,
     aoi
   });
 
+  const availableDomain: [Date, Date] | null = useMemo(
+    () => (start && end ? [start, end] : null),
+    [start, end]
+  );
+
+  const [brushRange, setBrushRange] = useState<[Date, Date] | null>(null);
+
+  useEffect(() => {
+    if (availableDomain && !brushRange) {
+      // TODO auto fit to available data? For now taking the whole user-defined range
```

Comment:
This could be tricky when there are different data domains, not sure it is need, but good of you to consider it! :)